src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.ts

```diff
@@ -59,7 +60,7 @@ export class RgwDaemonListComponent {
     );
   }
 
-  updateSelection(selection: CdTableSelection) {
+  updateSelection(selection: any) {
```

Comment:
To be consistent with all other components, we should remove this method and use the one provided by `ListWithDetails`.
```suggestion

```